recipes/mariadb-connector-c/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "mariadb-connector-c" %}
+{% set version = "3.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mariadb-corporation/mariadb-connector-c/archive/v{{ version }}.tar.gz  # [unix]
+  sha256: c0fda1fa6e52dc85de27156cd847088a72d40d9de6514f7efa57c8d93134a54c  # [unix]
+
+  url: https://github.com/mariadb-corporation/mariadb-connector-c/archive/v{{ version }}.zip  # [win]
+  sha256: b945adcdab6ea7369ceb082166cbb703ba1c98cb7d710fd5b6213c8b709e2e6a  # [win]
```

Are there any real differences in the sources? Or was it just b/c of the compression? If it was the compression you don't need this.

```suggestion
  url: https://github.com/mariadb-corporation/mariadb-connector-c/archive/v{{ version }}.tar.gz
  sha256: c0fda1fa6e52dc85de27156cd847088a72d40d9de6514f7efa57c8d93134a54c
```

---

There are no differences. I used another accepted recipe as an example.